Android.mk

```diff
@@ -67,7 +67,7 @@ LOCAL_PRELINK_MODULE := false
 LOCAL_MODULE := liboptee_cryptoki
 LOCAL_MODULE_TAGS := optional
 
-LOCAL_EXPORT_C_INCLUDE_DIRS := $(LOCAL_PATH)/public
+LOCAL_EXPORT_C_INCLUDE_DIRS := $(LOCAL_PATH)/libsks/include
```

Comment:
Actually I'm not sure about this but I think you're right. This change will export liboptee_cryptoki public APIs for use by other modules, instead of the libteec public APIs, right?

Reply:
Based on https://stackoverflow.com/questions/23106091/difference-between-local-export-c-includes-and-local-c-includes it should work, but my build to verify on hackbox is very slow (someone else hogging the cpu). Will post as update as soon as build completes.

Reply:
ok, then I propose I merge your change in optee_test without this modif in optee_client and postponed this "improvement".

Reply:
It works! :) Let's go with this as well and I'll remove the ugly include in etienne-lms/optee_test#1.